src/tests/statistics_tests.rs

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2021 ESR Labs GmbH. All rights reserved.
+//
+// NOTICE:  All information contained herein is, and remains
+// the property of E.S.R.Labs and its suppliers, if any.
+// The intellectual and technical concepts contained herein are
+// proprietary to E.S.R.Labs and its suppliers and may be covered
+// by German and Foreign Patents, patents in process, and are protected
+// by trade secret or copyright law.
+// Dissemination of this information or reproduction of this material
+// is strictly forbidden unless prior written permission is obtained
+// from E.S.R.Labs.
+#[cfg(test)]
+mod tests {
+    use crate::{
+        dlt::LogLevel,
+        statistics::{LevelDistribution, StatisticInfo},
+    };
+
+    fn get_stat_entities() -> Vec<(String, LevelDistribution)> {
+        vec![
+            (
+                String::from("Debug"),
```

Comment:
best not name it the same as the loglevel...it's meant to be a name for app-ids/context-ids/ecu-ids.
calling it "Debug" etc. is confusing